include/avif/internal.h

```diff
@@ -143,6 +143,7 @@ typedef struct avifReformatState
 // * AVIF_RESULT_OK              - Converted successfully with libyuv
 // * AVIF_RESULT_NOT_IMPLEMENTED - The fast path for this combination is not implemented with libyuv, use built-in YUV conversion
 // * [any other error]           - Return error to caller
+avifResult avifImageRGBToYUVLibYUV(avifImage * image, const avifRGBImage * rgb);
```

Comment:
I didn't know you were following the parameter order of `avifImageRGBToYUV()` in avif.h. Since `avifImageRGBToYUV()` is a public function, and it may be commonly used, let's not change it in this pull request. Please undo this change. Sorry about the extra work!